amxmodx/CTask.cpp

```diff
@@ -269,8 +269,11 @@ bool CTaskMngr::taskExists(int iId, AMX *pAmx)
 
 void CTaskMngr::startFrame()
 {
-	for (auto &task : m_Tasks)
+	auto lastSize = m_Tasks.length();
+	for(auto i = 0; i < lastSize; i++)
```

Comment:
`0u` was not a typo. `i` should be marked as 'unsigned', because `lastSize` has `size_t` type. You can cast `lastSize` to `int` as well but `u` is shorter.